examples/provider/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "xray" {
+  url = "${var.artifactory_url}/xray"
```

Comment:
You know, it's come up once or twice:

The current TF provider just takes the base URL and assumes `/xray/` or `/artifactory` and once or twice some places had remapped it with a proxy. I think we might consider something like this:

```terraform
provider "xray" {
  url = ".../foo"
  remapped = true
}
```

So, if `remapped = false` default, then we ignore anything not part of the host, but if it is remapped, then basically it overrides. 

We want the defaults to work for the vast majority of users - and the RT example does that - but we want an override

Reply:
we need to work on it later